installer/conf/container.conf

```diff
@@ -64,3 +64,17 @@
 <filter oms.container.**>
 	type filter_container
 </filter>
+
+#separate buffer for containers
+<match oms.container.**>
+  type out_oms
+  log_level info
+  buffer_chunk_limit 20m
+  buffer_type file
+  buffer_path /var/opt/microsoft/omsagent/state/state/out_oms_docker*.buffer
```

Comment:
![image](https://cloud.githubusercontent.com/assets/25937992/24264321/4729dcde-0fbd-11e7-9b4b-66805f211ee3.png)
 Yes, based on whatever path the omsagent uses right now.